### PR TITLE
Adding option to drop the table environment for latex writing

### PR DESCRIPTION
**Pull request by bsipocz**
In a few occasion I would have preferred to have the option to save out a table to latex format, but keeping only the ``tabular`` environment. Clearly editing the files and removing the lines manually is an option, but it's so much nicer if done by the writer itself.

I'll add tests and changelog when got the go ahead.

### CODE REVIEW

**Comment by hamogu** [comment]
As the original author of the latex writer I can say that this is totally cool. (Although I myself never had a use for that.) Go ahead!
It's a fairly simple PR, so I have no doubts that this will still make it into 2.0.

File: astropy/io/ascii/latex.py [hunk 1]
```diff
@@ -222,6 +224,8 @@ class Latex(core.BaseReader):
 
                 ascii.write(data, sys.stdout, Writer = ascii.Latex,
                             latexdict = {'tabletype': 'table*'})
+            If ``None``, the table environment will be dropped keeping only
```
Comment (taldcroft) [inline astropy/io/ascii/latex.py]: Put a comma after `dropped`.

**Comment by bsipocz** [comment]
(tests passed, but needed to rebase to resolve changelog conflict, thus the ci skip)

**Comment by bsipocz** [comment]
Comments are addresses, could someone hit the merge please?

**Comment by hamogu** [comment]
That will be @taldcroft. I don't have write / merge access for this repository.

**Pull request merged**